### PR TITLE
Publishing to Maven Central

### PR DESCRIPTION
**Pull request by williamboxhall**
- Change `build.gradle` to publish to Sonatype (which allows publishing to Maven Central
- Remove automatic deploys from Buildkite in favour or manual deploys
- Some basic CONTRIBUTING.md docs

https://search.maven.org/artifact/com.cultureamp/kestrel

I'll make another PR to update the README with installation instructions and the new logo

### CODE REVIEW

File: .buildkite/pipeline.yaml [hunk 1]
```diff
@@ -5,32 +5,3 @@ steps:
...
-    branches: "master"
-    command: bin/ci_deploy
-    agents:
-      queue: build-restricted
-
-  - name: build
-    branches: "!master"
-    command: bin/ci_build
-    agents:
-      queue: build-unrestricted
```
Comment (CurtisEppel) [inline .buildkite/pipeline.yaml:21]: Probably still want this step

**Review by CurtisEppel — DISMISSED** [review]
lgtm

File: build.gradle [hunk 2]
```diff
@@ -11,41 +11,90 @@ buildscript {
...
+                scm {
+                    connection = 'scm:git@github.com:cultureamp/kestrel.git'
+                    developerConnection = 'scm:git@github.com:cultureamp/kestrel.git'
+                    url = 'https://github.com/cultureamp/kestrel'
+                }
+
+                licenses {
+                    license {
+                        name 'The Apache License, Version 2.0'
+                        url 'http://www.apache.org/licenses/LICENSE-2.0.txt'
```
Comment (CurtisEppel) [inline build.gradle:70]: Should you check in a `LICENSE` file for the GitHub repo

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+We're still figuring out how this will look but for now if you need support, please raise an issue and we will get back
+to you within a few days.
+
+# Developing
+
+To compile and run tests
+
+`./gradlew test`
+
+If you're authorized to publish to nexus, which is a process done manually by a developer, you'll need to have been
```
Comment (CurtisEppel) [inline CONTRIBUTING.md:12]: Add a step to create a release tag and create a release?

File: build.gradle (hunk 2 above)
Comment (williamboxhall) [reply build.gradle:70]: Ah yeah I've seen that in other repo's - what's the reason for that over linking to the file this way?

File: CONTRIBUTING.md (hunk 3 above)
Comment (williamboxhall) [reply CONTRIBUTING.md:12]: oh, right 😬 I think I need to brush up a bit on the process. Maybe we could quickly pair on this later?

File: .buildkite/pipeline.yaml (hunk 1 above)
Comment (williamboxhall) [reply .buildkite/pipeline.yaml:21]: oh crap - yeah 🤦

File: CONTRIBUTING.md (hunk 3 above)
Comment (williamboxhall) [reply CONTRIBUTING.md:12]: I was also thinking maybe we wanna keep publishing to sonatype in the build as a manual step. WDYT?

File: build.gradle (hunk 2 above)
Comment (CurtisEppel) [reply build.gradle:70]: When you check in a LICENSE file it shows up in the about section on the Github repo. This is useful for someone who might want to fork the repository. GitHub also displays a nice overview of the license on the repository - https://github.com/ktorio/ktor/blob/master/LICENSE

File: CONTRIBUTING.md (hunk 3 above)
Comment (CurtisEppel) [reply CONTRIBUTING.md:12]: Yeah happy to pair. I have some other smaller open source suggestions

**Review by admackin — DISMISSED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (williamboxhall) [reply CONTRIBUTING.md:12]: I never got back to looking at this - @curtiseppel is it ok with you if we merge this and we loop back on doing proper release tags / descriptions at a later date when we have more time to give Kestrel love?

**Review by curtiseppel — APPROVED** [review]

**Pull request merged**